pkgs/build-support/setup-hooks/no-broken-symlinks.sh

```diff
@@ -45,6 +45,11 @@ noBrokenSymlinks() {
       symlinkTarget="$(realpath --no-symlinks --canonicalize-missing "$pathParent/$symlinkTarget")"
     fi
 
+    if [[ $symlinkTarget != "$NIX_STORE"/* ]]; then
+      nixInfoLog "symlink $path points outside the Nix store; ignoring"
```

Comment:
Does this print by default? Might be noisy if so. Probably fine though.

Reply:
It does not because the info level is fairly high.